scripts/hart_ip_types.zeek

```diff
@@ -88,8 +88,8 @@ export {
         session_log_record_client_i_pv6_address                                 : addr &log &optional;
         session_log_record_client_port                                          : count &log &optional;
         session_log_record_server_port                                          : count &log &optional;
-        session_log_record_connect_time                                         : count &log &optional;
-        session_log_record_disconnect_time                                      : count &log &optional;
+        session_log_record_connect_time                                         : time &log &optional; #changed from count to time
+        session_log_record_disconnect_time                                      : time &log &optional; #changed from count to time
```

Comment:
I'd remove these comments as well prior to release, it doesn't add anything of value for somebody reading the .zeek script